perf-object.go

```diff
@@ -59,13 +59,14 @@ type SpeedTestResult struct {
 
 // SpeedtestOpts provide configurable options for speedtest
 type SpeedtestOpts struct {
-	Size         int           // Object size used in speed test
-	Concurrency  int           // Concurrency used in speed test
-	Duration     time.Duration // Total duration of the speed test
-	Autotune     bool          // Enable autotuning
-	StorageClass string        // Choose type of storage-class to be used while performing I/O
-	Bucket       string        // Choose a custom bucket name while performing I/O
-	NoClear      bool          // Avoid cleanup after running an object speed test
+	Size                 int           // Object size used in speed test
+	Concurrency          int           // Concurrency used in speed test
+	Duration             time.Duration // Total duration of the speed test
+	Autotune             bool          // Enable autotuning
+	StorageClass         string        // Choose type of storage-class to be used while performing I/O
+	Bucket               string        // Choose a custom bucket name while performing I/O
+	NoClear              bool          // Avoid cleanup after running an object speed test
+	DisableContentSha256 bool          // Disable calculating sha256 for uploads
```

Comment:
EnableSha256

Reply:
We need to keep the current behavior as is to avoid breaking change in behavior.